Fixed #35502 -- Changed directory name of `mysite` in tutorial. (#18235)
django/django: docs/intro/tutorial01.txt

application-specific settings.

From the command line, ``cd`` into a directory where you'd like to store your
code and create a new directory named ``djangotutorial``.

.. console::

   $ mkdir djangotutorial

Then, run the following command to bootstrap a new Django project:

.. console::

   $ django-admin startproject mysite djangotutorial

> This will create your project in the ``djangotutorial`` directory.  Its
> name doesn't matter to Django; you can rename it to anything you like. If it didn't

Reviewer: ```suggestion
This will create a project called ``mysite`` in the ``djangotutorial`` directory. 
```

I would leave the `Its name doesn't matter to Django...` where it was.

.. console::

>    $ django-admin startproject mysite djangotutorial

Reviewer 2: Perhaps this was meant to be the following?

`django-admin startproject djangotutorial`

Reviewer 2: Ah, never mind. My mistake in understanding how this command works.

> This will create a ``djangotutorial`` directory in your current directory. If it didn't

Reviewer 3: Per my testing, this is no longer true. When running the command above without having an existing folder named `djangotutorial`, I get this error:

```
$ django-admin startproject mysite djangotutorial
CommandError: Destination directory '/home/nessita/djangotutorial' does not exist, please create it first.
```

I think this intro needs to be adjusted to guide the user creating a `djangotutorial` folder as prerequisite so they don't get frustrated.

Author: Thank you for your observation. I have changed the text.

These files are:

* :file:`djangotutorial/`: A root directory that contains your project. Its

>   name doesn't matter to Django; you can rename it to anything you like.

Reviewer 4: Is this bullet needed and is ```Let's look at what :djadmin:`startproject` created:` showing `djangotutorial/``` still valid?
The command no longer created the directory so we might want to adjust this. Maybe moving the "name doesn't matter to Django" further up to when we created the directory

Author: Moved it up to wen we created the drectory.

Reviewer 3: Thank you @petmetic! Did you commit/push this change? I'm not seeing it :thinking:

Author: Pushed it now. ad a bit of trouble and had to rebase it... sorry. 
Hope everything is ok now. Please let me know if I there is anything else.

Reviewer 3: @sarahboyce I have re-reviewed the latest version and I think your comment is spot on. Since `djangotutorial` was created by the person following the tutorial, is not "created by `startproject`". I have tweaked and refactored things a bit. Thanks!

* :file:`manage.py`: A command-line utility that lets you interact with this
  Django project in various ways. You can read all the details about
  :file:`manage.py` in :doc:`/ref/django-admin`.

> * :file:`mysite/`: A child directory with the actual Python package for your

Reviewer 4: ```suggestion
* :file:`mysite/`: A child directory that is the actual Python package for your
```

These files are:

> * The outer :file:`djangotutorial/` root directory is a container for your project.

Reviewer: I'd revert the "Its name doesn't matter..." as per comment above.

Author: Thank you for reading my changes again. :)
